encord/orm/project.py

```diff
@@ -149,10 +149,76 @@ class ProjectDataset:
     pass
 
 
-class ProjectCopyOptions(Enum):
+class ProjectCopyOptions(str, Enum):
     COLLABORATORS = "collaborators"
     DATASETS = "datasets"
     MODELS = "models"
+    LABELS = "labels"
+
+
+class ReviewApprovalState(str, Enum):
+    APPROVED = "APPROVED"
+    PENDING = "PENDING"
+    REJECTED = "REJECTED"
+    DELETED = "DELETED"
+    NOT_SELECTED_FOR_REVIEW = "NOT_SELECTED_FOR_REVIEW"
+
+
+class CopyDatasetAction(str, Enum):
+    ATTACH = "ATTACH"
+    CLONE = "CLONE"
+
+
+@dataclass
+class CopyDatasetOptions:
+    """Options for copying the datasets associated with a project."""
+
+    action: CopyDatasetAction = CopyDatasetAction.ATTACH
+    """
+    One of `CopyDatasetAction.ATTACH` or `CopyDatasetAction.CLONE`. (defaults to ATTACH)
+    ATTACH: Attach the datasets associated with the original project to the copy project.
+    CLONE: Clone the data units from the associated datasets into a new dataset an attach it to the copy project.
```

Comment:
ah interesting. I'd just keep the docstrings up there. Basically closest to where it is defined, just in case we need it in multiple places.